required_fields.node.js

```diff
@@ -177,14 +177,18 @@ module.exports = function (homeValues) {
         });
     }
     for (let system of ['1', '2']) {
-        if ([null, undefined, ''].indexOf(homeValues['heating_fuel_'+system]) === -1 &&
-            ['', 'none', null, undefined, 'baseboard', 'wood_stove'].indexOf(homeValues['heating_type_'+system]) === -1 &&
-            !(homeValues['heating_fuel_'+system] === 'electric' && homeValues['heating_type_'+system] === 'central_furnace')
+        let heatingType = homeValues['heating_type_'+system];
+        let heatingFuel = homeValues['heating_fuel_'+system];
+        let heatingEfficiencyMethod = homeValues['heating_efficiency_method_'+system];
+
+        if ([null, undefined, ''].includes(heatingFuel) &&
+            ['', 'none', null, undefined, 'baseboard', 'wood_stove'].includes(heatingType) &&
+            !(heatingFuel === 'electric' && ['central_furnace', 'boiler'].includes(heatingType))
```

Comment:
This is the only line changed in a way that affects the behavior of the code - every other line is only changed by the new variables.